pkg/helpers/rates.go

```diff
@@ -10,17 +10,6 @@ var (
 	// Default Rate
 	DefaultRate = vegeta.Rate{Freq: 5, Per: time.Second}
 
-	// Cluster Services
-	CreateClusterRate = vegeta.Rate{Freq: 10, Per: time.Second}
-	ListClustersRate  = vegeta.Rate{Freq: 10, Per: time.Second}
-
-	// Account Services
-	SelfAccessTokenRate   = vegeta.Rate{Freq: 1000, Per: time.Hour}
-	ListSubscriptionsRate = vegeta.Rate{Freq: 2000, Per: time.Hour}
-	GetCurrentAccountRate = vegeta.Rate{Freq: 6, Per: time.Minute}
-
 	// Authorization Services
-	AccessReviewRate            = vegeta.Rate{Freq: 100, Per: time.Second}
-	RegisterNewClusterRate      = vegeta.Rate{Freq: 1000, Per: time.Hour}
 	RegisterExistingClusterRate = vegeta.Rate{Freq: 25, Per: time.Second}
```

Comment:
I did not find a usage for this should I keep it in the meantime @kurtismullins

Reply:
Yes, please leave this one in for now. We need to implement a test which essentially runs the "Register New Clusters" but for clusters with an existing UUID. This rate is used for that test.